9.2-2.3/alpine/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM postgres:9.2-alpine
+MAINTAINER Régis Belson <me@regisbelson.fr>
+
+ENV POSTGIS_VERSION 2.3.1
+ENV POSTGIS_SHA256 0f1e0fe08f55b6db791d8600bb343e4f77590725acbbec99094a30b8b7439cd8
+
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
```

Comment:
Do you mind trying to get it to work with [repository pinning](http://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Repository_pinning)? If we can ensure that only the `testing` packages are installed from the testing repo, I'd be happy with this change.

Thanks for your patience ❤️